content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -270,6 +271,18 @@ to the IP of the your WSL instance (`ip addr show dev eth0`) (For example,
 installed, follow the official 
 [Kubernetes on Windows instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
 
+## k3ai [*Alhpa*]
+
+K3ai is a lightweight infrastructure in a box specifically built to install and configure AI tools and platforms to quickly experiment over laptops and edge devices.
+
+Is main goal is to provide the most simple way to install Kubeflow pipelines without having to worry about setting up requirements and dependencies.
```

Comment:
```suggestion
K3ai's main goal is to provide a quick way to install Kubernetes (K3s-based) and Kubeflow Pipelines with NVIDIA GPU support and TensorFlow Serving with just one line. (For Kubeflow and other component support, check [K3ai's website](https://kf5ai.gitbook.io/k3ai/#components-of-k-3-ai) for updates.)
```

- I think "simplest" is subjective, but we can use demonstrate that it's simple by showing it takes one line to get it "up and running". LMKWYT
- The K3ai's official docs mention which components are supported, so I added them here.